registers/x86.xml

```diff
@@ -0,0 +1,685 @@
+<?xml version='1.0' encoding='utf-8'?>
```

Comment:
Just  to verify, do these files come from the upstream repo?

Reply:
Unfortunately not. I generated these files by hand using https://wiki.osdev.org/CPU_Registers_x86 , https://wiki.osdev.org/CPU_Registers_x86-64 , and a few wikipedia pages for the SIMD registers. I tried to follow the general form of the upstream repo's files,  but they were generated independently for this PR. If this is too much of an issue I can try looking into developing a more automated means to generate the files like the upstream repo does. :)

Reply:
No worries. It would be good  to have a script to do this automatically at some point but  I don't see these files changing often so no big deal